Assets/_BForBoss/_Analytics/Scripts/PerigonAnalytics.cs

```diff
@@ -4,21 +4,25 @@
 
 namespace BForBoss
 {
+    #region STRUCTS
     public readonly struct Event
     {
         public const String PlayerDeath = "Player - Death";
+        public const String CheckpointReached = "Checkpoint - Reached";
     }
 
     public readonly struct EventAttribute
     {
         public const String Course = "course";
         public const String Name = "name";
+        public const String Time = "timer";
```

Comment:
can you name it "time"

Reply:
I can't name it to time because when I used time, mixpanel overwrites it for some reason. Was not sure what else to change it to. I could try removing the sessions?

Reply:
ah that's unfortunate. that's okay then.